ArduPlane/GCS_Mavlink.cpp

```diff
@@ -109,16 +109,18 @@ MAV_STATE GCS_MAVLINK_Plane::system_status() const
 }
 
 
-void Plane::send_attitude(mavlink_channel_t chan)
+void GCS_MAVLINK_Plane::send_attitude() const
 {
+    const AP_AHRS &ahrs = AP::ahrs();
+
     float r = ahrs.roll;
-    float p = ahrs.pitch - radians(g.pitch_trim_cd*0.01f);
+    float p = ahrs.pitch - radians(plane.g.pitch_trim_cd*0.01f);
```

Comment:
I did not know this was a thing, but it's a terribly nasty hack... Do we actually have people using this?

Reply:
How can they *not* be using it?  It's a critical telemetry message, after all!

Reply:
Oh not your part, the plane parameter of pitch_trim_cd is terrible...

Reply:
Fixed in #18134